samples/renode_example/README.md

```diff
@@ -0,0 +1,102 @@
+# Renode Example
+
+[Renode](https://renode.io/) is an open-source software development framework
+with commercial support from Antmicro that lets you develop, debug and test
+multi-node device systems reliably, scalably and effectively.
+
+The goal of this sample is to review the Renode platform and how to use it with
+the nrf52840dk board.
+
+This sample contains a simple firmware for the nrf52840dk board, which does the
+following:
+
+- On the press of a Button 1 (on the nrf52840DK) it prints
+  `GPIO Interrupt fired`,
+- It has enabled GPIO and Sensor shells and
+- It specifies in its device tree LIS2DW12 accelerometer and STTS751 temperature
+  sensors.
```

Comment:
```suggestion
- On the press of Button 1 (on the nrf52840DK) it prints
  `GPIO Interrupt fired`.
- It has enabled GPIO and Sensor shells.
- It specifies in its device tree LIS2DW12 accelerometer and STTS751 temperature
  sensors.
```
If the text of your bullet point is a complete sentence (or multiple sentences), use capital letters and punctuation. If your points are not structured as proper sentences, you don’t need to end with punctuation.

Reply:
Corrected :+1: